jessica_hart/assignments/Portfolio/index.html

```diff
@@ -0,0 +1,100 @@
+                            <a href="#">Continue reading...</a>
+                        </div>
+                    </div>
+                </div><!-- end of top content -->
+                <div id="sub_content">
+                    <div class="container">
+                        <div class="desc">
+                            <h2>La Mode</h2>
+                            <img src="images/lamode.png" alt="La Mode, Ecommerce website">
+                            <p>La Mode is an Ecommerce website designed to market various clothing products. Users are able to view the available garments, select their desired quantity, and compile a shopping cart for making a final purchase.</p>
+                        </div>
+                        <h3>Technologies:</h3>
+                        <div class="tech">
+                            <img src="images/php.png" alt="php">
+                            <img src="images/codeigniter.png" alt="codeigniter">
+                            <img src="images/jQuery.png" alt="jQuery">
+                            <img src="images/ajax.png" alt="ajax">
+                            <img src="images/mysql.png" alt="mysql">
+                            <img src="images/HTML.png" alt="HTML">
+                            <img src="images/CSS.png" alt="CSS">
+                            <img src="images/bootstrap.png" alt="bootstrap">
+                            <br>
+                            <span class="button"><a href="#">La Mode</a></span>
+                        </div>
+                    </div>
+                    <div class="container">
+                        <div class="desc">
+                            <h2>Family Contacts</h2>
+                            <img src="images/familycontacts.png" alt="Family Contacts, relative family management">
+                            <p>Family Contacts is a free application for managing, sharing, and visualizing your family relationships for both extended and immediate family. As an essential feature of the project, your log-in information determines who you can see and reveals how people are related to you.</p>
+                        </div>
+                        <h3>Technologies:</h3>
+                        <div class="tech">
+                            <img src="images/JavaScript.png" alt="JavaScript">
+                            <img src="images/mongodb.png" alt="mongodb">
+                            <img src="images/expressjs.png" alt="expressjs">
+                            <img src="images/angular.png" alt="angular">
+                            <img src="images/nodejs.png" alt="nodejs">
+                            <img src="images/HTML.png" alt="HTML">
+                            <img src="images/CSS.png" alt="CSS">
+                            <img src="images/socket.png" alt="socket">
+                            <br>
```

Comment:
Thanks for the response! Sadly I had left that while trying to fix the green buttons, the same issue you had on the board too it seems (when you had to separate the buttons by wrapping in their own div). The <br> tags have been removed now.